db/test/test_stats.py

```diff
@@ -296,14 +296,27 @@ def add_empty_lowlevel(mbid, lossless, date):
     gid_type = gid_types.GID_TYPE_MSID
```

Comment:
would it be possible to use `db.data.write_low_level` for the body of this method? e.g. we create the empty data object, and then call write_low_level to write it to the database instead of duplicating this code. I guess we added it here ourselves to skip a few tests. I think it makes sense to unify this code if possible.

Reply:
I've moved the `get_submission_offset` function to the top level of the `db.data` module and renamed it, more appropriately, to `get_next_submission_offset`. I'm looking into using `db.data.write_low_level` entirely here, but one issue I'm finding is that this function requires a unique sha256 field. In `db.data.write_low_level` the data_sha256 field is found by hashing the encoded data_json, but data_json will be the same for every item added by this function. I know I can construct the data before passing it to `db.data.write_low_level` like so:

```data = {"data": {}, "metadata": {"audio_properties": {"lossless": lossless}, "version": {"essentia_build_sha": "sha1"}}}```

But how can I ensure that the sha256 will be unique for each item added by this function? Or how can I differentiate the data field for each one when they need to be empty?